staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -361,6 +362,20 @@ func (s *Scheme) AddGeneratedConversionFuncs(conversionFuncs ...interface{}) err
 	return nil
 }
 
+// AddConversionFunc registers a function that converts between a and b by passing objects of those
+// types to the provided function. The function *must* accept objects of a and b - this machinery will not enforce
+// any other guarantee.
+func (s *Scheme) AddConversionFunc(a, b interface{}, fn conversion.ConversionFunc) error {
```

Comment:
a bit confusing to have `AddConversionFuncs` for typed and `AddConversionFunc` for untyped.